src/components/application_manager/src/application_impl.cc

```diff
@@ -31,6 +31,7 @@
  */
 
 #include "application_manager/application_impl.h"
+#include <iomanip>
```

Comment:
@EKuliiev if you find out that this header should be included, I would do it with separate commit

Reply:
I use it there:
https://github.com/smartdevicelink/sdl_core/pull/2167/commits/c7da899496a38e14f81e91e02389613313af62e8#diff-b301fcf652ff21d656f6378b9ba22658R237